charms/kserve-controller/requirements.in

```diff
@@ -2,3 +2,4 @@ charmed-kubeflow-chisme
 lightkube
 lightkube-models
 ops
+pyyaml==6.0.1
```

Comment:
I believe this was introduced by 09515b97e476e182da6628793bc060536a759a97, but I think we should not pin pyyaml here. This was related to an issue with python-libjuju that was resolved in latest versions.